static/styles/dark_theme.css

```diff
@@ -328,7 +328,15 @@
     .new-style .button {
         background-color: hsla(0, 0%, 0%, 0.2);
 
-        &:not(.sea-green, .btn-danger, .btn-warning, .btn-link, .poll-vote) {
+        &:not(
+                .sea-green,
+                .btn-danger,
+                .btn-warning,
+                .btn-link,
+                .poll-vote,
+                button.poll-option,
+                button.add-task
+            ) {
```

Comment:
This `not` block is really messy, and we should figure out a way to refactor it (probably involving these buttons not having the `new-style .button` class applied to them), but that's an existing mess and probably best left for future work.